### PR TITLE
ENH: Improve slicer.util.exit documentation

### PR DESCRIPTION
**Pull request by lassoan**
Fixes https://issues.slicer.org/view.php?id=4470

### CODE REVIEW

File: Base/Python/slicer/util.py [hunk 1]
```diff
@@ -12,7 +12,25 @@ def quit():
...
+
+  .. code-block:: python
+
+    import sys
+    sys.exit(status)
+
+  """
+
   from slicer import app
+  # Prevent automatic exit overwrite this manually set exit code
```
Comment (jcfr) [inline Base/Python/slicer/util.py]: this manually set exit code -> this manually to set exit code

Comment (jcfr) [reply Base/Python/slicer/util.py]: That said, I am not sure of the meaning of "overwrite this to manually set exit code"

**Comment by lassoan** [comment]
I mean overwriting the exit code that is set in this call

File: Base/Python/slicer/util.py (hunk 1 above)
Comment (jcfr) [reply Base/Python/slicer/util.py]:
```
# Prevent automatic exit and ensure the "status" exit code passed to the following
# "slicer.exit()" call is considered
```

Comment (lassoan) [reply Base/Python/slicer/util.py]: how about this?

    Prevent automatic exit (for example, triggered by "--testing") overwrite the exit code that we set now

Comment (jcfr) [reply Base/Python/slicer/util.py]: `..  overwrite` -> `.. and overwrite ...`

**Comment by jcfr** [comment]
Closing. Fix integrated in [r27220](http://viewvc.slicer.org/viewvc.cgi/Slicer4?view=revision&revision=27220) and [r27224](http://viewvc.slicer.org/viewvc.cgi/Slicer4?view=revision&revision=27224)

**Comment by lassoan** [comment]
Thank you!

**Pull request closed without merging**